src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Applications.cshtml

```diff
@@ -16,139 +16,157 @@
+                            {
+                                foreach (var app in Model.Applications)
+                                {
+                                    <tr class="govuk-table__row" data-table-row="one">
+                                        <td class="govuk-table__cell sorting_1">
+                                            @if (app.Status == SFA.DAS.LevyTransferMatching.Domain.Types.ApplicationStatus.Pending)
+                                            {
+                                                <div class="govuk-checkboxes govuk-checkboxes--small" das-highlight-error-for="@app.EncodedApplicationId" error-class="govuk-form-group--error">
+                                                    <div class="govuk-checkboxes__item">
+                                                        <input class="govuk-checkboxes__input" id="application_select_@app.EncodedApplicationId" error-class="govuk-input--error"  name="ApplicationsToReject" type="checkbox" value="@app.EncodedApplicationId">
+                                                        <label class="govuk-label govuk-checkboxes__label"> </label>
+                                                    </div>
+                                                </div>
+                                            }
+                                         </td>
+                                        <td class="govuk-table__cell"><a href="applications/@app.EncodedApplicationId" class="govuk-link">@app.DasAccountName</a></td>
+                                        <td class="govuk-table__cell">
+                                            @app.DisplayAmount
+                                        </td>
+                                        <td class="govuk-table__cell">@app.Duration months</td>
+                                        <td class="govuk-table__cell">
+                                            <criteria-list application="@app"></criteria-list>
+                                        </td>
+                                        <td class="govuk-table__cell"><strong class="@app.Status.GetCssClassForSender()">@app.Status.GetLabelForSender()</strong></td>
+                                    </tr>
+                                }
+                            }
+
+                            else
+                            {
+                                <tr class="govuk-table__row" data-table-row="one">
+                                    <td colspan="6" class="govuk-table__cell">No applications received yet</td>
+                                </tr>
+                            }
+                         }
+                      </tbody>
+                 </table>
+              </div>  
+                <div class="govuk-grid-row">
+                    <div class="govuk-grid-column-full">
+                        <section class="das-highlight">
+                            <div class="govuk-button-group">
```

Comment:
The padding on this panel looks wrong compared to the screenshot - could you check it? @jamesmking might be able to help.

Reply:
I discussed it with James and he has provided the correct HTML. All looks good now.